.gitcommitmsg

```diff
@@ -1,3 +1,3 @@
-[CS|EX|WF] Fix|Update|New:
+[CS|CF|EX|WF] Fix|Update|New:
```

Comment:
What does CF stand for?

Reply:
Config File... Wanted to add further files for vim tmux... in the future